src/it/properties-file-support/verify.groovy

```diff
@@ -1,25 +1,6 @@
-import groovy.json.JsonSlurper
-
 import java.nio.file.Paths
 
-import static ru.yandex.qatools.allure.report.TestHelper.checkReportDirectory
-import static ru.yandex.qatools.allure.report.TestHelper.getTestCases
+import static io.qameta.allure.maven.TestHelper.checkReportDirectory
 
 def outputDirectory = Paths.get(basedir.absolutePath, 'target', 'site', 'allure-maven-plugin')
 checkReportDirectory(outputDirectory, 1)
-
-def dataDirectory = outputDirectory.resolve('data')
```

Comment:
makes sense to fix the test (how else you can check that properties file processed correctly?)